sdk/python/feast/feature_server.py

```diff
@@ -83,30 +82,21 @@ def shutdown_event():
     @app.post("/get-online-features")
     def get_online_features(body=Depends(get_body)):
         try:
-            # Validate and parse the request data into GetOnlineFeaturesRequest Protobuf object
-            request_proto = GetOnlineFeaturesRequest()
-            Parse(body, request_proto)
-
+            body = json.loads(body)
```

Comment:
I'm curious why they serialized this into Protobuf....it was probably on purpose so I'm not exactly sure changes it to Python objects is ideal but if tests pass I suppose it's hard to argue.